features/logging/kibana.feature

```diff
@@ -92,6 +92,13 @@ Feature: Kibana related features
     When I run the :check_log_count web action
     Then the step should succeed
     """
+    # Verify the token are encrypted in kibana logs 
+    Given the first user is cluster-admin
+    Given a pod becomes ready with labels:
+      | logging-infra=kibana |
+    When I run the :logs client command with:
+      | resource_name | <%= pod.name %> |
```

Comment:
@anpingli I think we need point out the container name here as there are 2 containers in kibana pod, or the step will fail
```
When I run the :logs client command with:
      | resource_name | <%= pod.name %> |
      | c                        | xxxxxxxxxxxxxx       |
```